cmd/lotus-miner/actor.go

```diff
@@ -1028,6 +1167,115 @@ var actorConfirmChangeWorker = &cli.Command{
 	},
 }
 
+var actorConfirmChangeBeneficiary = &cli.Command{
+	Name:      "confirm-change-beneficiary",
+	Usage:     "Confirm a beneficiary address change",
+	ArgsUsage: "[minerAddress]",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "really-do-it",
+			Usage: "Actually send transaction performing the action",
+			Value: false,
+		},
+		&cli.BoolFlag{
+			Name:  "existingBeneficiary",
+			Usage: "send confirmation from the existing beneficiary address, defaults to the new beneficiary address",
+		},
+	},
+	Action: func(cctx *cli.Context) error {
+		if !cctx.Args().Present() {
+			return fmt.Errorf("must specify one argument")
+		}
+
+		api, acloser, err := lcli.GetFullNodeAPI(cctx)
+		if err != nil {
+			return xerrors.Errorf("getting fullnode api: %w", err)
+		}
+		defer acloser()
+
+		ctx := lcli.ReqContext(cctx)
+
+		maddr, err := address.NewFromString(cctx.Args().First())
+		if err != nil {
+			return xerrors.Errorf("parsing beneficiary address: %w", err)
+		}
+
+		mi, err := api.StateMinerInfo(ctx, maddr, types.EmptyTSK)
+		if err != nil {
+			return xerrors.Errorf("getting miner info: %w", err)
+		}
+
+		if mi.PendingBeneficiaryTerm == nil {
+			return fmt.Errorf("no pending beneficiary term found for miner %s", maddr)
+		}
+
+		fmt.Println("Confirming Pending Beneficiary Term of:")
+		fmt.Println("Beneficiary: ", mi.PendingBeneficiaryTerm.NewBeneficiary)
+		fmt.Println("Quota:", mi.PendingBeneficiaryTerm.NewQuota)
+		fmt.Println("Expiration Epoch:", mi.PendingBeneficiaryTerm.NewExpiration)
+
```

Comment:
I think we need to ask the user for confirmation here -- if not, we could see that the confirmed change we're making isn't what we want, but not be able to do anything except go NOOOOOOO

Reply:
We have a check for `--really-do-it` below

Reply:
I'll put it right after these print statements to make it clearer that that's what is intended

Reply:
So you're saying you want users to run the command without `--really-do-it`, get to these prints, then have the command "fail" because `--really-do-it` wasn't provided, and then have them rerun with `--really-do-it`?

I think that's reasonable -- I would change the `--really-do-it` failure message to say something please re-run with the flag after reviewing details CAREFULLY".